db/1.9/modules/ROOT/pages/database-stored-procedure.adoc

```diff
@@ -0,0 +1,211 @@
+			]]]>
+		</db:input-parameters>
+	</db:stored-procedure>
+----
+
+
+== Configure the Input Parameters Field in the Stored Procedure Operation
+
+In the Input parameters field, specify values that create a map in which keys are the name of an input parameter to be set on the JDBC prepared statement. Reference each parameter in the SQL text using a colon prefix, for example, `where id = :myParamName`. The map’s values contain the actual assignation for each parameter.
+
+To configure the Stored Procedure operation:
+
+. In your Studio flow, select the *Stored procedure* operation.
+. In the operation configuration screen, set the *SQL Query Text* field to the query to execute, for example:
++
+`{ call updatePlanetDescription('Venus', :description) }`
++
+. Set the *Input parameters* field to `{'description' : payload}`.
+
+The following screenshot shows the configuration:
+
+.Stored procedure with Input parameter configuration
+image::database-stored-example-1.png[Stored procedure with Input parameter configuration]
+
+In the XML editor, the `<db:stored-procedure>` and `<db:input-parameters>` configuration looks like this:
+
+[source,xml,linenums]
+----
+<db:stored-procedure config-ref="dbConfig">
+    <db:sql>{ call updatePlanetDescription('Venus', :description) }</db:sql>
+    <db:input-parameters>
+        #[{'description' : payload}]
+    </db:input-parameters>
+</db:stored-procedure>
+----
+== Configure the In Out Parameters Field in the Stored Procedure Operation
+
+In the In Out Parameters field, specify values that create a map in which keys are the name of a parameter to be set on the JDBC prepared statement which is both input and output. Reference each parameter in the SQL text using a colon prefix, for example, `where id = :myParamName`. The map’s values contain the actual assignation for each parameter.
+
+To configure the Stored Procedure operation:
+
+. In your Studio flow, select the *Stored procedure* operation.
+. In the operation configuration screen, set the *SQL Query Text* field to the query to execute, for example:
++
+`{ call updatePlanetDescription('Venus', :description) }`
++
+. Set the *In out parameters* field to `Edit inline`.
+. Click the plus sign (*+*) to add a new value.
+. In the new window, set the *Key* field to `myInt` and the *Value* field to `3`.
+
+The following screenshot shows the configuration:
+
+.Stored procedure with In Out parameters configuration
+image::database-stored-example-2.png[Stored procedure with In Out parameters configuration]
+
+In the XML editor, the `<db:stored-procedure>` and `<db:in-out-parameter>` configuration looks like this:
+
+[source,xml,linenums]
+----
+<db:stored-procedure config-ref="dbConfig">
+  <db:sql>{ call doubleMyInt(:myInt) }</db:sql>
+  <db:in-out-parameters>
+      <db:in-out-parameter key="myInt" value="3"/>
+  </db:in-out-parameters>
+</db:stored-procedure>
+----
+
+== Configure the Input and Output Parameters Fields in the Stored Procedure Operation
```

Comment:
replace with `Configure the Output Parameters Field in the Stored Procedure Operation` for consistency with the prior titles.